refresh-data.sh

```diff
@@ -55,6 +55,12 @@ check_data() {
 refresh_data() {
     echo 'Importing refresh db'
     gunzip < "$refresh_dump_name" | cfgov/manage.py dbshell > /dev/null
+    SCHEMA="$(gunzip -c $refresh_dump_name | grep -m 1 'CREATE SCHEMA' | sed 's/CREATE SCHEMA \(.*\);$/\1/')"
+    if [ "${PGUSER}" != "${SCHEMA}" ]; then
```

Comment:
With this change it becomes necessary to have `PGUSER` defined in the environment, which to date has not yet been the case with the non-Docker [virtual environment setup](https://cfpb.github.io/consumerfinance.gov/running-virtualenv/) (note that currently [in `.env_SAMPLE`](https://github.com/cfpb/consumerfinance.gov/blob/aae64a1ffd60b5063ba58811d4f4606869b950c5/.env_SAMPLE#L57-L66) the PG environment variables are only optional).

If `PGUSER` is not defined, the `DROP SCHEMA ...` statement fails and then so does the whole refresh script. Could this line be modified to only do this step if `PGUSER` is defined? Something like `if [ -z "${PGUSER}" ] && [ ... ]`?

Reply:
Good point, wonder if it would be better to just have `PGUSER` default to `cfpb` if not defined within `refresh-data.sh`?

`${PGUSER:-cfpb}`

not sure what makes most sense here.

Reply:
@chosak after talking with @willbarton, we decided that setting a default that matches the default database user we have in code (`cfpb`), would be the most reliable. We also only ever expect sql dumps to have schema `cfpb`. 

If a user was trying to load an sql dump where schema was not `cfpb`, we would expect them to also set `PGUSER`. 

Does this make sense and work for you? I have made the change for you to test out as well.

Reply:
Thanks @jslay-excella! Your new change works perfectly for me without `PGUSER` set.